apps/design-system/src/components/ServiceInfoBadge.stories.tsx

```diff
@@ -16,6 +16,18 @@ export const HTTP = {
   }
 }
 
+export const Server = {
+  args: {
+    protocol: 'server'
```

Comment:
We should probably rename `protocol` to `info` or something similar since it's not a Protocol Badge anymore. What do you think?

Reply:
`info` would be a good choice, or we could use `serviceName` or `serviceInfo`, which would be a bit clearer.

Reply:
Sure :)